src/dev/mod.ts

```diff
@@ -135,8 +131,53 @@ export default manifest;
   await Deno.writeTextFile(manifestPath, manifestStr);
   console.log(
     `%cThe manifest has been generated for ${routes.length} routes and ${islands.length} islands.`,
-    "color: blue; font-weight: bold",
+    "color: blue; font-weight: bold"
+  );
+}
+
+interface IMapper {
+  dir: string;
+  file: string;
+}
+export async function routeWarnings(directory: string) {
+  const manifest = await collect(directory);
+  const { routes } = manifest;
+
+  // iterate each route
+  const mapped = routes.map((route: string): IMapper => {
+    const parsed = pparse(route);
+    return { dir: parsed.dir, file: parsed.base };
+  });
+
+  const routesPerDepth = mapped.reduce(
+    (totals: any, p) => ({ ...totals, [p.dir]: (totals[p.dir] || 0) + 1 }),
+    {}
   );
+
+  const errors = () => {
+    const output = [];
+    for (const [key, value] of Object.entries(routesPerDepth)) {
+      const ones = mapped.filter((p) => {
+        return (
+          p.dir === key && routesPerDepth[key] > 1 && p.file.match(/\[(.*?)\]/g)
+        );
+      });
+      if (ones.length > 0) {
+        output.push(`routes${key}`);
+      }
+    }
+    return output;
+  };
+
+  const errorsList = errors();
+  if (errorsList.length > 0) {
+    console.log(
+      `%cCheck ${errorsList} for potential routing issues.
+You may have dynamic and static routes overwriting each other.
+Please check the documentation for more information. http://localhost:8000/docs/getting-started/dynamic-routes`,
```

Comment:
You are pointing to your `localhost:8000` here, replace it with the Fresh website.

Reply:
But when they are developing it will always be  running locally here in the terminal?